bucket/determined.json

```
@@ -0,0 +1,21 @@
+{
+    "version": "0.26.7",
+    "description": "The Determined CLI is a command line tool that lets you launch new experiments and interact with a Determined cluster.",
+    "homepage": "https://github.com/sirredbeard/determined-windows-cli",
+    "license": "Apache-2.0",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/sirredbeard/determined-windows-cli/releases/download/0.27.0/determined-cli.zip",
+            "hash": "EF8B24D00AAE8C2019B7C80E1626CB69AAD03070E88E0F5055BA5CEDC16BE2EC"
+        }
+    },
+    "bin": "det.exe",
+    "checkver": "github",
+    "autoupdate": {
+        "url": "https://github.com/sirredbeard/determined-windows-cli/releases/download/$version/determined-cli.zip",
+        "hash": {
+            "url": "https://github.com/sirredbeard/determined-windows-cli/releases/download/$version/determined-cli.zip.sha256",
+            "type": "sha256"
+        }
```

```suggestion
        "url": "https://github.com/sirredbeard/determined-windows-cli/releases/download/$version/determined-cli.zip"
        }
```

This will fail since no hash files are available in release artifacts.
Either provide hash file(s) along with binaries or remove the `hash` section (scoop will calculate actual sha256 using file itself).